src/challengeserver/views.py

```diff
@@ -59,6 +85,11 @@ def get(self, request):
 
 
 class SysinfoView(APIView):
+    """
+    Get system information.
```

Comment:
Probably worth clarifying, this is getting the system info of the andromeda node, not the box. Also, worth noting for us to investigate at a later point - how does andromeda in docker handle sysinfo, I don't think its going to be at all accurate.